shapes/contacts/group-index.shex

```diff
@@ -1,7 +1,20 @@
 PREFIX vcard: <http://www.w3.org/2006/vcard/ns#>
-PREFIX xsd: <https://www.w3.org/2001/XMLSchema#>
+PREFIX xsd:   <https://www.w3.org/2001/XMLSchema#>
+
+# This shape describes part of the information we store when adding a group to
+# an address book. The resource it is used for is an index of groups. There will
```

Comment:
I would say "The resource it is used for is an index of groups", since it could be reused in a million different ways by other people. You could just add that as an _example_ of where it _could_ be used, but if so I'd suggest rewording it slightly to something like "could be used to represent a group within an index of groups" (but then what's the difference between "an index of groups" and "a collection of groups", or "a container containing groups"...?).